install/install-oasis

```diff
@@ -49,22 +50,32 @@ cp $SCRIPTDIR/aufs.postupdate /opt/etc/osupdate.d
 /opt/etc/osupdate.d/aufs.postupdate
 
 # create srv directory if it isn't there yet
-SRVDIR="/net/nas01/srv/`uname -n|cut -d. -f1`"
-if [ ! -d $SRVDIR ]; then
-    mkdir $SRVDIR
-    setfacl -b $SRVDIR
-    chmod 755 $SRVDIR
-    mkdir $SRVDIR/cvmfs
-    mkdir $SRVDIR/spool
+NETSRVDIR="/net/vm08/oasis-share/srv/$MYSHORTNAME"
+if [ ! -d $NETSRVDIR ]; then
+    mkdir $NETSRVDIR
+    setfacl -b $NETSRVDIR
+    chmod 755 $NETSRVDIR
 fi
 
+# mount srv directory permanently at /srv
+if ! grep -q "[ \t]/srv[ \t]" /etc/fstab; then
+    echo "`echo $NETSRVDIR|sed 's,/net/\([^/]*\),\1:,'` /srv	nfs 	defaults 	0 0" >>/etc/fstab
```

Comment:
Add a comment with an example output of this line as the bash can be tricky to read.